uwblueprint/sistering
# Add edit posting page

## Ticket link

Closes #514 


## Implementation description
* Changed updatePosting resolver to generate bulk shifts and create them if posting in question is in draft (allows shifts to be overwritten)
* Added code in backend to determine recurrence interval based on existing shifts
* Added cascade delete for shift to signups
* Added route for editing posting
* Reworked create posting pages + forms to support create/update mode


## Steps to test
1. Go to the admin homepage
2. Try editing any posting in draft
3. Updating the draft either as draft or published should be reflected in the admin homepage
4. For editing posting not in draft, user should not be able to edit existing shifts or save these posting as drafts (we use duplicate action instead)
5. Updating posting not in draft should reflect in admin homepage


## What should reviewers focus on?
* Correctness
* Code cleanliness

## Checklist
